front/css/app.css

```
@@ -1846,7 +1856,7 @@ textarea[readonly],
 
 #modal-ok
 {
-    z-index: 1051; /*highest priority*/
+    z-index: 11051; /*highest priority*/
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)